### PR TITLE
Vbproj support

### LINKED ISSUE #74: Vbproj support?
Do you plan to support vb projects?
There should not be a lot of difference I guess..

### PR DESCRIPTION
**Pull request by hvanbakel**
Fixes #74.

I don't know if either of you guys has a bunch of vb or F# projects lying around. I don't which was why I never wanted to put it into master.

I guess we can commit it as 'experimental'...

We would probably want to create a separate tool, maybe as dotnet-migrate-2017 so you could do:
`dotnet migrate-2017` 
or something similar. That might also address #162

### CODE REVIEW

**Comment by hvanbakel** [comment]
disclaimer, this is the first time ever I did a rebase, so hopefully that was done correctly 😉

**Comment by mungojam** [comment]
I've not used vb.net before either or much fsharp.

For rebasing like this I tend to just use VS and then you wouldn't change the commit message to mention rebasing. The idea of this type of rebasing is to make it look like history was linear, like you just started on VB projects support today, so first commit would be 'add vbproj support'

File: Project2015To2017/ProjectConverter.cs [hunk 1]
```diff
@@ -52,25 +59,21 @@ public IEnumerable<Project> Convert(string target)
...
-						yield return this.ProcessFile(file, null);
-						break;
-
-					default:
+					foreach (var project in ConvertSolution(target))
+					{
+						yield return project;
+					}
+				}
+				else if (ProjectFileMappings.TryGetValue(extension, out var fileExtension))
```
Comment (andrew-boyarshin) [inline Project2015To2017/ProjectConverter.cs]: `switch` (with `when`) would still seem nicer, but that's not a real issue to be fixed.

**Comment by hvanbakel** [comment]
Thanks for the review, I need to make more use of the fancy new features
indeed.

I won't merge prior to 2.0 being out though.

**Pull request merged**